### PR TITLE
docs: clarify shallow clone behavior

### PR DESCRIPTION
**Pull request by navierula**

### CODE REVIEW

File: www/content/ci.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ source.developers.google.com/p/YourProjectId/r/github-YourGithubUser-YourGithubR
 you're building off.
 
 This repo has the wrong name, so to prevent Goreleaser from publishing to
-the wrong github repo, put in the your .goreleaser.yml file's release section:
+the wrong github repo, add your .goreleaser.yml file's release section:
```
Comment (caarlos0) [inline www/content/ci.md]:
```suggestion
the wrong github repo, add to your `.goreleaser.yml` file's release section:
```

**Comment by caarlos0** [comment]
hey, just a small suggestion :)

**Comment by caarlos0** [comment]
thanks for the PR BTW :)

**Comment by navierula** [comment]
No problem. :)

**Comment by caarlos0** [comment]
Thanks :D

**Pull request merged**